application/static/swagger.yml

```diff
@@ -289,3 +434,147 @@ components:
           type: string
         rating:
           type: integer
+        rating_image:
+          type: string
+    Charities:
+      properties:
+        count:
+          type: integer
+        charities:
+          type: array
+          items:
+            $ref: '#/components/schemas/Charity'
+  examples:
```

Comment:
Remainder of the file is adding data to display examples of data in requests. Previously was displaying schema instead.